CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+* When submitting a feature request:
+    * Explain why you need the feature &mdash; what's your use-case, what's your domain.
+    * Explaining the problem you face is more important than suggesting a solution.
+      Report your issue even if you don't have any proposed solution.
+    * If there is an alternative way to do what you need, then show the code of the alternative.
+
+## Submitting PRs
+
+We love PRs. Submit PRs [here](https://github.com/Kotlin/dataframe/pulls).
+However, please keep in mind that maintainers will have to support the resulting code of the project,
+so do familiarize yourself with the following guidelines.
+
+* All development (both new features and bug fixes) is performed in the `master` branch.
+    * Base PRs against the `master` branch.
+    * PR should be linked with the issue,
+      excluding minor documentation changes, the addition of unit tests, and fixing typos.
+* If you make any code changes:
+    * Follow the [Kotlin Coding Conventions](https://kotlinlang.org/docs/reference/coding-conventions.html).
+    * [Build the project](#building) to make sure it all works and passes the tests.
+* If you fix a bug:
+    * Write the test the reproduces the bug.
+    * Fixes without tests are accepted only in exceptional circumstances if it can be shown that writing the
+      corresponding test is too hard or otherwise impractical.
+* If you introduce any new public APIs:
+    * All new APIs must come with documentation and tests.
+    * If you plan API additions, then please start by submitting an issue with the proposed API design
+      to gather community feedback.
+    * [Contact the maintainers](#contacting-maintainers) to coordinate any big piece of work in advance via submitting an issue.
+* If you fix documentation:
+    * If you plan extensive rewrites/additions to the docs, then please [contact the maintainers](#contacting-maintainers)
+      to coordinate the work in advance.
+    * Also, we have a special simple [guide](https://github.com/Kotlin/dataframe/blob/master/docs/contributions.md) how to contribute in the documentation.
+
+## PR workflow
+
+0. Contributor build the library locally and run all unit test via Gradle task `dataframe:test`
+   (see the ["Building"](#building) chapter).
+1. Contributor submits the PR if the local build is successful and tests are green.
+2. Reviewer put his name to the "Reviewers" section of the proposed PR at the start of the review process.
+3. Reviewer leaves the comments or marks the PR with the word "LGTM."
+4. Contributor answers the comments or fixes the proposed PR.
+5. Reviewer marks the PR with the word "LGTM."
+6. Maintainer could suggest merging the master branch to the PR branch a few times due to changes in the `master` branch.
+7. Maintainer runs TC builds (unit tests and examples as integration tests).
+8. The TC writes the result (passed or not passed) to the PR checks at the bottom of the proposed PR.
```

Comment:
No "The" ;)

Reply:
@zaleslaw you forgot this one. "The TeamCity" is incorrect.

Reply:
Interesting, the Grammarly left this as a part of the text

Reply:
Because it doesn't know what the abbreviation stands for :) It doesn't know it's a name

Reply:
Thanks, got it, will fix directly in the master